### PR TITLE
Add SetNodeOperatorNames Simple DVT factory support 

### PR DESCRIPTION
**Pull request by katamarinaki**
### Description

This PR adds a support for Simple DVT module factory `SetNodeOperatorNames`.

### Checklist:

- [x] Checked the changes locally.
- [ ] Affects / requires changes in other services (Matomo / Sentry / CloudFlare / etc.).

### CODE REVIEW

File: modules/motions/ui/MotionFormStartNew/Parts/StartNewSDVTNodeOperatorNamesSet.tsx [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+
+                <Fieldset>
+                  <InputControl
+                    name={`${fieldNames.nodeOperators}.${i}.newName`}
+                    label="New Name"
+                    disabled={isFieldDisabled}
+                    rules={{
+                      required: 'Field is required',
+                      validate: (value: string | undefined) => {
+                        if (!value?.length) {
```
Comment (BATMAH69) [inline modules/motions/ui/MotionFormStartNew/Parts/StartNewSDVTNodeOperatorNamesSet.tsx:194]: why required option is not enough?

File: modules/motions/ui/MotionFormStartNew/Parts/StartNewSDVTNodeOperatorNamesSet.tsx [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+
+                        if (
+                          typeof idInNameMap === 'number' &&
+                          idInNameMap !== currentId
+                        ) {
+                          return 'Name must be unique'
+                        }
+
+                        if (maxNodeOperatorNameLength?.lt(value.length)) {
+                          return `Name must be less than ${maxNodeOperatorNameLength.toString()} characters`
```
Comment (BATMAH69) [inline modules/motions/ui/MotionFormStartNew/Parts/StartNewSDVTNodeOperatorNamesSet.tsx:217]: does `.toString()` required here?

File: modules/motions/ui/MotionFormStartNew/Parts/StartNewSDVTNodeOperatorNamesSet.tsx [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+        }
+      }
+
+      for (const nodeOperator of selectedNodeOperators) {
+        result[parseInt(nodeOperator.id)] = nodeOperator.newName
+      }
+
+      const invertedRecord: Record<string, number | undefined> = {}
+      for (const key in result) {
+        if (result.hasOwnProperty(key)) {
```
Comment (BATMAH69) [inline modules/motions/ui/MotionFormStartNew/Parts/StartNewSDVTNodeOperatorNamesSet.tsx:109]: Could you add this in other for-in loops

**Review by BATMAH69 — APPROVED** [review]

File: modules/motions/ui/MotionFormStartNew/Parts/StartNewSDVTNodeOperatorNamesSet.tsx (hunk 2 above)
Comment (katamarinaki) [reply modules/motions/ui/MotionFormStartNew/Parts/StartNewSDVTNodeOperatorNamesSet.tsx]: No it's not, I'll delete it

**Pull request merged**